osidb/models.py

```diff
@@ -2088,15 +2088,21 @@ def is_community(self) -> bool:
             name=self.ps_module, ps_product__business_unit="Community"
         ).exists()
 
+    # TODO unlike SFM2 this is mostly unused. Is that intentional or did something get missed?
+    #      - see e.g. backends/jira/rest/request.py in SFM2
+    # TODO unlike SFM2, SLA can't (?) evaluate the new streams part as long as this is a property
```

Comment:
@osoukup unlike SFM2, SLA can't (?) evaluate the new streams part of is_compliance_priority as long as this is a property on affect (since affect doesn't have streams). Should we file a new Jira to fix it?

(This is sth I identified during work on stream preselection for fedramp, but should be filed as a separate Jira if a valid concern)

Reply:
Yes, most probably. Added a comment to that Jira.